dev.pulsar_edit.Pulsar.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Have you submitted this file upstream?

Reply:
I have now (https://github.com/pulsar-edit/pulsar/pull/380) but it won't be ready.